service-api/src/main/java/greencity/constant/ErrorMessage.java

```diff
@@ -65,6 +65,8 @@ public final class ErrorMessage {
     public static final String EMPLOYEE_IS_NOT_ASSIGN =
         "Employee service could not be assign for order or this is manager which assign managers";
     public static final String SERVICE_IS_NOT_FOUND_BY_ID = "couldn't found service with id: ";
+    public static final String SERVICE_IS_NOT_FOUND_BY_TARIFF_ID = "couldn't found service for tariff with id: ";
```

Comment:
Should the message start with a capital letter?

Reply:
Changed